src/Collection/CollectionTrait.php

```diff
@@ -421,22 +421,25 @@ public function combine($keyPath, $valuePath, $groupPath = null)
      * {@inheritDoc}
      *
      */
-    public function nest($idPath, $parentPath)
+    public function nest($idPath, $parentPath, $childrenKey = null)
     {
         $parents = [];
         $idPath = $this->_propertyExtractor($idPath);
         $parentPath = $this->_propertyExtractor($parentPath);
         $isObject = true;
+        if (empty($childrenKey)) {
```

Comment:
PHP will complain if default value of `$childrenKey` is set to `'children'`,  since it's null in the interface right?